apps/aecore/src/aec_fork_block_settings.erl

```diff
@@ -344,7 +348,15 @@ extra_accounts_json_file() ->
     "extra_accounts_test.json".
 
 contracts_json_file() ->
```

Comment:
Isn't this just a subset of the function below? I can't remember why we wan't the TEST one?

Reply:
This filename is a legacy one ... back then one could only set accounts (and later contracts) via this file. The suffix "_test" I believe comes from the original usage for tests only, but at some point all "custom" network ID's has been using this.

Reply:
Ah, I see what your comment is about now .. nvm

Reply:
I've moved the functions out of test and made a generic function for creating the file name.